packages/main/src/components/add-prefix-modal/AddPrefixModal.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect, useState } from "react";
+import styles from "../add-prefix-modal/ChannelPrefixes.module.css";
+import { getCurrentWorkspace } from "../add-prefix-modal/utils/Common";
+import axios from "axios";
+
+const AddPrefixModal = ({ closeModal, onAdd }) => {
+  const orgID = getCurrentWorkspace();
+  const token = sessionStorage.getItem("token");
+  axios.defaults.headers.common["Authorization"] = `Bearer ${token}`;
+  const [data, setData] = useState({
+    description: "",
+    prefix: "",
+    creator: "Admin"
+  });
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    axios
+      .patch(`https://api.zuri.chat/organizations/${orgID}/prefixes`, data)
+      .then(res => {
+        const responseData = res.data;
+        if (responseData.status === 200) {
+          onAdd(data);
+          closeModal(null)(false);
+        }
+      })
+      .catch(err => {
+        console.error(err);
```

Comment:
the user is not opening the console
handle it with a toast notification or something else

Reply:
There's already an alert for it